src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
Fine with me then :) then my changes to convert bool returns to Result<(), BoolError> could also get in without headaches.
Are there other breaking changes on your list?